lib/t/location.ex

```diff
@@ -13,6 +13,10 @@ defmodule T.Location do
       {:ok, %{"location" => %{"latitude" => lat, "longitude" => lon}}} ->
         [lat, lon]
 
+      {:error, :database_unknown} ->
+        Logger.error("location database_unknown}")
+        [55.755516, 37.615040]
+
```

Comment:
I explicitly didn't handle that error since it's a setup error and should be make known to sentry. I think with the current setup Logger.error would only appear in cloudwatch logs.